src/manager.js

```diff
@@ -13,24 +13,102 @@ import './images/turing-logo.png'
 
 //date
 let manager = new Manager();
+$('#future-bookings').hide();
 
-$('.date').html(manager.todaysDate());
 
 let bookingData = fetch('https://fe-apps.herokuapp.com/api/v1/overlook/1904/bookings/bookings')
   .then(data => data.json())
   .then(data => data.bookings)
 
-
 let roomData = fetch('https://fe-apps.herokuapp.com/api/v1/overlook/1904/rooms/rooms')
   .then(data => data.json())
   .then(data => data.rooms)
 
-Promise.all([bookingData, roomData]).then((requiredData) => {
-  manager.rooms = requiredData[1];
+let userData = fetch('https://fe-apps.herokuapp.com/api/v1/overlook/1904/users/users')
+  .then(data => data.json())
+  .then(data => data.users)
+
+Promise.all([bookingData, roomData, userData]).then((requiredData) => {
   manager.booked = requiredData[0];
-  let bookedTodayData = manager.bookings();
-  $('.revenue').html(manager.totalSpend(bookedTodayData));
+  manager.rooms = requiredData[1];
+  manager.users = requiredData[2];
+  let bookedTodayData = manager.bookings(manager.todaysDate());
+  $('.date').html(manager.todaysDate());
+  $('.revenue').html(manager.getTotalSpend(bookedTodayData));
   $('.percentage').html((bookedTodayData.length/manager.rooms.length) * 100)
-    console.log(manager.availableToday(bookedTodayData))
   manager.availableToday(bookedTodayData).forEach(room => {$('.available').append(`<span class="rooms">Room:${room.number} Beds: ${room.numBeds} ${room.bedSize.toUpperCase()} Price: $${room.costPerNight}</span>`)})
 }).catch(data => console.log('Fetch error', data))
+
+$('#search').keyup(() => {
+  $('#search-results').html('')
+  manager.customerSearch($('#search').val()).forEach(customer => {
+    $('#search-results').append(`<span id="${customer.id}" class="upcoming-rooms">Name: ${customer.name}</span>`)
+  })
+})
```

Comment:
Is the manager.customerSearch() method returning more than one user object?  
The forEach is throwing me off a little if its always just returning one user.

Reply:
I did try to do it for case, but I missed it in one part and just added it to the other. Really great catch, helped me with a bug fix I didnt event know I had.